provider/blob/src/blob_schema.rs

```diff
@@ -168,7 +177,7 @@ impl Default for BlobSchemaV2<'_> {
     }
 }
 
-impl<'data> BlobSchemaV2<'data> {
+impl<'data, LocaleVecFormat: VarZeroVecFormat> BlobSchemaV2<'data, LocaleVecFormat> {
     pub fn load(&self, key: DataKey, req: DataRequest) -> Result<&'data [u8], DataError> {
```

Comment:
Thought: It would be nice if `load` didn't have to monomorphize, but I don't see an easy way to prevent that. The function is pretty thin anyway; most of the work is in `impl Write for ZeroTriSimpleAsciiCursor` and `impl Writeable for DataLocale` which hopefully should remain thier own functions.